lib/openstack_query/query_blocks/query_builder.py

```diff
@@ -104,17 +119,20 @@ def parse_where(
             )
 
         preset_handler = self._get_preset_handler(preset, prop)
-        self.client_side_filter = preset_handler.get_filter_func(
+        client_side_filter = preset_handler.get_filter_func(
             preset=preset,
             prop=prop,
             prop_func=prop_func,
             filter_func_kwargs=preset_kwargs,
         )
 
-        self.server_side_filters = self._server_side_handler.get_filters(
+        server_side_filters = self._server_side_handler.get_filters(
             preset=preset, prop=prop, params=preset_kwargs
         )
-        if not self.server_side_filters:
+        if server_side_filters and not isinstance(server_side_filters, list):
+            server_side_filters = [server_side_filters]
```

Comment:
Do this in the setter, user's of the class should get something sane back according to the type we promised